hotsos/core/ycheck/engine/common.py

```diff
@@ -77,41 +78,23 @@ def plugin_defs(self):
 
 class YHandlerBase(object):
```

Comment:
ABC and ABCMeta are basically the same but used/defined in different ways (see https://docs.python.org/3/library/abc.html). I use ABC as a rule but I think they are interchangeable. The reason why neutron uses ABCMeta might be historical because ABC was added after ABCMeta in python 3.4.